tests/__tests__/auto-cleanup.js

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/vue'
+import '@testing-library/jest-dom/extend-expect'
+
+const HelloWorldComponent = {
+  template: `<h1>Hello World</h1>`
+}
+// This just verifies that by importing VTL in an
+// environment which supports afterEach (like jest)
+// we'll get automatic cleanup between tests.
+test('render the first component', () => {
+  const { container } = render(HelloWorldComponent)
+
+  const h1 = container.querySelector('h1')
+  expect(h1.textContent).toEqual('Hello World')
+})
+
+const NewComponent = {
+  template: `<h1>Automatic clean after each test</h1>`
+}
+
+test('it should cleanup after each test and render NewComponent', () => {
+  render(NewComponent)
+  expect(document.body.innerHTML).toEqual(
```

Comment:
I believe we could keep this a bit simpler. Instead of rendering a new component, we could simply expect that the `<body>` is empty:

```js
expect(document.body.innerHTML).toEqual('')
```